admin/test/test_release.py

```diff
@@ -1177,6 +1184,13 @@ def get_dispatcher(self):
         # given RPMs, not that any metadata files are copied.
         self.assertEqual(expected_files, set(files_on_s3.keys()))
 
+        # The repository is build in self.packages_directory
+        # Ensure that that does not leak into the metadata.
+        packages_gz = files_on_s3[os.path.join(self.target_key, 'Packages.gz')]
```

Comment:
'build' -> 'built'
Reference the issue that prompted this test: FLOC-1823 :question: